src/components/TopicCard/TopicCard.tsx

```diff
@@ -4,10 +4,11 @@ import '../Card/Card.css';
 interface TopicCard {
   title: string
   image: string
-  id: number
+  id: string
   sentiment: number
   updateUserSentiment: (userSentiment: number) => void
   selectedTopic: string
+  updateHistory: (localHistory: string[]) => void;
```

Comment:
This is added here, but it is not included in the TopicCard Component. Do you want this to run onClick in line 32?

Reply:
Also, I think I missed this in the past PR, but how is TopicCard different than Card? Is the TopicCard component necessary?

Reply:
@nzambonivergara Topic Card is different than Card in that it has a specific URL that includes the topic and the id in the URL     `to={`/feed/${selectedTopic}/${id}`}`

Reply:
Yes. Fixed.